tests/integration/targets/ios_user/tests/cli/auth.yaml

```diff
@@ -37,6 +37,21 @@
       ansible.builtin.assert:
         that:
           - results.failed
+
+    - name: Test long password
+      cisco.ios.ios_user:
+        name: auth_user
+        privilege: 15
+        role: network-operator
+        state: present
+        configured_password: "{{ 'a' * 128 }}"
+      ignore_errors: true
```

Comment:
this won't pass lint, igore_errors is deprecated. Please run ansible-lint locally with production profile